src/form/elements/Select3.ts

```diff
@@ -53,6 +53,11 @@ export interface ISelect3Options<T extends Readonly<IdTextPair>> {
    * @default false
    */
   required: boolean;
+  /**
+   * readonly flag
+   * @default false
+   */
+     readonly: boolean;
```

Comment:
Indentation is slightly off. 

This PR is fine for me, but I would not invest much more time in it. We are going to refactor tdp_comments to use `react-select` at some point, which makes most of the adjustments obsolete.